ray_on_golem/server/cluster/nodes.py

```diff
@@ -256,6 +266,25 @@ async def _create_activity(self) -> Tuple[Activity, str, str]:
 
         return activity, ip, ssh_proxy_command
 
+    async def _get_agreement(self) -> Agreement:
+        if self._priority_manager_stack:
+            self._add_state_log("[1/9] Getting agreement from providers in priority subnet...")
```

Comment:
but honesly, I would rather stay with "Getting agreement..." because, from the user's perspective, this is just this step where our aim is to acquire an agreement with a provider who supplies the head node -> we can explain in the error message after the timeout that we've failed to sign an agreement with the providers from the priority subnet and that we're retrying the step with the regular nodes... but otherwise, having this content of the first step is too much information for a regular user...